src/offat/__main__.py

```diff
@@ -115,8 +115,8 @@ def start():
     )
     parser.add_argument(
         '-s',
-        '--ssl',
-        dest='ssl',
+        '--ssl_verify',
```

Comment:
let's keep the tag simple to `--ssl`, dest variable seems fine.

Reply:
Is it enforcing ssl or ssl verification 

I suggest to avoid confusion to not call it "ssl" when it's enabling / disabling verification

Reply:
then we can rename it to `--ssl-verify`

Reply:
Great idea